docs/content/tutorials/firewall.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: doc_page
+---
+
+What to Do When You Have a Firewall
+-----------------------------------
+When you are behind a firewall, the Maven Druid dependencies will not be accessible, as well as the IRC wikipedia channels that feed realtime data into Druid. To workaround those two challenges, you will need to:
+
+1. make the Maven Druid dependencies available offline
+2. make the Wikipedia example GeoLite DB dependency available offline
+3. use an alternative approach to load data into Druid
+
+## Making Maven Druid Dependencies Available Offline
```

Comment:
Maybe this should just link to the FAQ or maybe this and the FAQ should both link to some other place. It's the same information being sought from two different angles.